src/data/utils.js

```diff
@@ -107,17 +128,19 @@ function zip(a, b) {
  * @returns {CorrelationMetric}
  */
 export function generateCorrelationMetrics(response, explanatory) {
-  const response_values = response.map((row) => row.value);
-  const explanatory_values = explanatory.map((row) => row.value);
+  const zippedEpiData = zipEpiDataRow(response, explanatory);
```

Comment:
what happens if there is no overlap or no given data?, e.g. if we have no data for a selected region?

Reply:
Good question.  If there's not at least 28*2 shared elements we should return a message saying there wasn't enough data.